models/ed/R/model2netcdf.ED2.R

```diff
@@ -643,11 +647,25 @@ put_T_values <- function(yr, nc_var, out, lat, lon, begins, ends, ...){
   
   
   # ----- define ncdf dimensions
-  
+  # setup output time and time bounds
+  sub_date_range <- seq(begins, by = "day", length.out = length(lubridate::yday(begins):ends))
+  sub_jdates <- lubridate::yday(lubridate::floor_date(seq(sub_date_range[1], sub_date_range[length(sub_date_range)],length.out= length(out[[1]]))))
```

Comment:
Can you give this and `sub_date_range` more informative variable names, just to make it clearer what's going on here? I _think_ I follow, but it's hard to wade through all the parentheses.

From just a readability standpoint, you could also consider piping (`%>%`) some of this to cut back on nested parentheses. Then, you could add a line comment to each step of the logic.

Reply:
Also, see above comment about `along.with`.